components/QuizCardPage/NextButton/index.js

```diff
@@ -23,4 +23,5 @@ export const StyledButtonNext = styled.button`
   font-size: 16px;
   cursor: pointer;
   margin-top: 20px;
+  margin-bottom: 2px;
```

Comment:
Is 2px a value the user can really see? Or did you want to write 20px instead?

Reply:
i changed this to 10px